core/domain/translation_domain_test.py

```diff
@@ -23,6 +23,279 @@
 from core.tests import test_utils
 
 
+class TranslatableObject1(translation_domain.BaseTranslatableObject):
```

Comment:
These names (1, 2, 3, 4, 5) are not easy to parse. Try to use names that, when read, help the reader remember what the class is without having to look it up. It should be possible to just read the test implementations and for those to make sense "on the surface" without needing to look up the detailed implementation of the TranslatableObject dummy classes.

Reply:
Updated.